src/app/containers/Recommendations/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { string } from 'prop-types';
+import 'isomorphic-fetch';
+import webLogger from '#lib/logger.web';
+
+const logger = webLogger();
+
+const RecommendationsContainer = ({ path }) => {
+  const [recommendationsData, setRecommendations] = useState({ items: [] });
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(
+          `https://camino-broker-cdn.api.bbci.co.uk${path}`,
+        );
+        const data = await response.json();
+        setRecommendations(data);
+      } catch (error) {
+        logger.error(`HTTP Error: "${error}"`);
+      }
+    };
+    fetchData();
+  }, [path]);
+  const { items } = recommendationsData;
+  return (
+    !!items.length && (
```

Comment:
Doesn't the double negation here cancel each other out, or am I reading this check wrong?